mmv1/third_party/terraform/website/docs/d/oracle_database_db_nodes.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Oracle Database"
+description: |-
+  List all database nodes of a Cloud VmCluster.
+---
+
+# google_oracle_database_db_nodes
+
+List all DbNodes of a Cloud VmCluster.
+
+For more information see the
+[API](https://cloud.google.com/oracle/database/docs/reference/rest/v1/projects.locations.cloudVmClusters.dbNodes).
+
+## Example Usage
+
+```hcl
+data "google_oracle_database_db_nodes" "my_db_nodes"{
+	location = "us-east4"
+	cloud_vm_cluster = "vmcluster-id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cloud_vm_cluster` - (Required) The ID of the VM Cluster.
+
+* `location` - (Required) The location of the resource.
+
+- - -
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+* `db_nodes` - (Output) List of dbNodes. Structure is [documented below](#nested_dbnodes).
+
+<a name="nested_dbnodes"></a> The `db_nodes` block supports:
+
+* `name` - User friendly name for the resource.
+
+* `properties` - Various properties of the database node. Structure is [documented below](#nested_properties).
+
+<a name="nested_properties"></a> The `properties` block supports:
+
+* `ocid`- OCID of database node.
+
+* `ocpu_count` - OCPU count per database node.
+
+* `memory_size_gb` - The allocated memory in GBs on the database node.
+
+* `db_node_storage_size_gb` - The allocated local node storage in GBs on the database node.
+
+* `db_server_ocid` - The OCID of the Database server associated with the database node.
+
+* `hostname` - The host name for the database node.
+
+* `state` - State of the database node.
+<a name="nested_states"></a>Allowed values for `state` are:<br>
```

Comment:
Nit: "Possible" might be better since they're output values (so from the user perspective "allowed" is not meaningful.)

```suggestion
<a name="nested_states"></a>Possible values for `state` are:<br>
```